pkg/cli/debug/debug.go

```diff
@@ -338,6 +341,11 @@ func (o *DebugOptions) Complete(cmd *cobra.Command, f kcmdutil.Factory, args []s
 		return err
 	}
 
+	o.RBACClient, err = rbacv1client.NewForConfig(config)
```

Comment:
if you can make a namespace, you don't need this binding.

Reply:
right, thanks - removed it!